cmd/opdoc/opdoc.go

```diff
@@ -389,6 +407,21 @@ func main() {
 	integerConstantsTableMarkdown(constants)
 	constants.Close()
 
+	named := []namedType{}
```

Comment:
```suggestion
	named := make([]namedType, 0, len(logic.AllStackTypes))
```
Of course, I don't really think performance matters here, I'd just prefer we use this pattern everywhere, for consistency with places where it does matter.